IronSoftware.Drawing/IronSoftware.Drawing.Common/AnyBitmap.cs

```diff
@@ -478,13 +489,13 @@ public AnyBitmap(Uri Uri)
         /// <param name="Uri">The uri of the image.</param>
         /// <returns></returns>
         /// <seealso cref="AnyBitmap"/>
-        /// <seealso cref="FromUri"/>
-        public static AnyBitmap FromUri(Uri Uri)
+        /// <seealso cref="FromUriAsync"/>
+        public static async Task<AnyBitmap> FromUriAsync(Uri Uri)
```

Comment:
This is a breaking change, do we have any way to have the old signature still working?

Reply:
Good catch, I'll rollback the FromUri function and mark deprecate.